src/sandstorm/util.c++

```diff
@@ -95,6 +96,47 @@ kj::Maybe<kj::AutoCloseFd> raiiOpenAtIfExists(
   }
 }
 
+// scratch buffer for file paths; used by raiiOpenAtIfExistsContained:
+thread_local char path_buf[PATH_MAX+1];
```

Comment:
I'd allocate this on the stack instead of `thread_local`. `thread_local`'s implementation is rather complex, especially for large values like this.